posts/tests/test_models.py

```diff
@@ -1,9 +1,9 @@
 from django.forms import ValidationError
 from django.test import TestCase
 from django.contrib.auth import get_user_model
-from posts.models import Post, Comment
+from posts.models import Post, Comment, Like
 
-from .constants import COMMENT_DATA, POST_DATA
+from .constants import COMMENT_DATA, POST_DATA, LIKE_DATA
```

Comment:
Please revert these changes as well

Reply:
Done. Getting rid of some migrations actually is giving me some interdependency errors